src/pcs/c1/PageMods.js

```diff
@@ -56,6 +57,12 @@ const setMulti = (document, settings, onSuccess) => {
   if (settings.scrollTop !== undefined) {
     Scroller.setScrollTop(settings.scrollTop)
   }
+  if (settings.setTextSizeAdjustmentPercentage !== undefined) {
```

Comment:
For consistency sake let's drop the `set` prefix and call this just `textSizeAdjustmentPercentage`.
Also add this to the list of fields in the comment on line 37.

Reply:
Done.